webrtc-stats.html

```diff
@@ -1143,9 +1143,20 @@
              boolean                       readable;
              unsigned long long            bytesSent;
              unsigned long long            bytesReceived;
-             double                        roundTripTime;
+             double                        averageRtt;
+             double                        currentRtt;
```

Comment:
Thanks for the clarification. I think currentRtt still makes sense in case
the intervals are polling interval is shorter than Ta or seconds.
On Sat, 2 Jul 2016 at 22:29, Taylor Brandstetter <notifications@github.com>
wrote:

> In webrtc-stats.html
> <https://github.com/w3c/webrtc-stats/pull/54#discussion_r69381797>:
>
> > @@ -1143,9 +1143,20 @@
> >               boolean                       readable;
> >               unsigned long long            bytesSent;
> >               unsigned long long            bytesReceived;
> > -             double                        roundTripTime;
> > +             double                        averageRtt;
> > +             double                        currentRtt;
>
> Well, the value of "Ta" (the time between connectivity checks) is up to
> the implementation. And it also depends on how many pairs you have and
> other factors. So it would be difficult to predict when connectivity checks
> are sent.
>
> Consent checks, however, are recommended to be sent and random time
> intervals around 5 seconds, and the interval MUST be no shorter than 4
> seconds.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/webrtc-stats/pull/54/files/9fca1cea4e9131b1043ab463d12b72cfae15c636#r69381797>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAXVkgsqp5SrU6QSRfIw7goyT45yghBXks5qRrv8gaJpZM4IxleG>
> .
>